src/components/Inline-quotes/index.js

```diff
@@ -24,8 +24,8 @@ text-align: center;
     }
 
     h4 {
-        text-align: right;
-        flex: 0 0 65%;
+        text-align: ${props => props.onlyQuoteIsPresent ? "center" : "right"};
```

Comment:
Have you confirmed that this doesn't negatively impact other quotes?

Reply:
Yes, I have confirmed that all the other quotes are as they were before.

Reply:
Wonderful. 🙂